restcomm/restcomm.ussd/src/main/java/org/restcomm/connect/ussd/telephony/UssdCallManager.java

```diff
@@ -282,7 +282,7 @@ private ActorRef outbound(final Object message) throws ServletParseException {
         SipURI to = (SipURI)sipFactory.createSipURI(request.to(), uri);
 
         String transport = (to.getTransportParam() != null) ? to.getTransportParam() : "udp";
-        from = outboundInterface(transport);
+        //from = outboundInterface(transport);
```

Comment:
@maria-farooq that is correct, we could also remove the private method. We already have a check in the API for USSDpush so that the From field cannot be null. 

example:

curl -X POST -H "application/json" http://ACae6e420f425248d6a26948c17a9e2acf:MYPWD@192.168.112-04-24/Accounts/ACae6e420f425248d6a26948c17a9e2acf/UssdPush  -d "To=111111" -d "Url=http://192.168.1.3:8080/restcomm-rvd/services/apps/AP6c234396213e437aa2f8e55c424134ca/controller"
From can not be null.